.travis.yml

```diff
@@ -65,7 +65,7 @@ install:
   - spack clean -a
   - source /etc/profile &&
     source $SPACK_ROOT/share/spack/setup-env.sh
-  - spack load cmake
+  - spack load cmake@3.15.0%gcc@7.5.0
```

Comment:
why did you introduce this change with this pull request?

Reply:
was answered with #3305

Reply:
Yes, was trying out the fix yesterday, as this PR encountered the issue #3304. But you are right, I should have marked the PR as WIP then.

Reply:
Rebased.